### DISCUSSION TITLE
Arc mark

### DISCUSSION THREAD

**Discussion by marcorezende** [uwdata/mosaic#552] [Q&A]
Hello guys, i recently discovered mosaic and I am new to data visualization world.

In my scenario i need to have one arc/pie/donut chart. I was unable to find one mark like that in the docs. Is there a way that i have a chart like that or it is not supported?

**Comment by jheer** [comment]
Unfortunately vgplot has no support for pie charts or arc marks at the moment. vgplot is a proof-of-concept library that builds on top of [Observable Plot](https://observablehq.com/plot/features/marks), which does not have an arc mark at present.

One could instead build a custom Mosaic client, but obviously that requires significantly more work.

**Reply by marcorezende** [reply]
Thanks for answer, i was manage to do that combining vega and mosaic. 😀
